draft-ietf-sedate-datetime-extended.md

```diff
@@ -196,16 +196,61 @@ Internet Date/Time Format:
 Timestamp:
 : An unambiguous representation of some instant in time.
 
+UTC Offset:
+: An amount of time before or after UTC that defines the relationship
+  between UTC and the local time in a particular place.
+
 Z:
 : A suffix which, when applied to a time, denotes a UTC offset of
   00:00; often spoken "Zulu" from the ICAO phonetic alphabet
   representation of the letter "Z" (from {{Section 2 of RFC3339}}).
 
 Time Zone:
-: A time zone that is included in the Time Zone Database (often
-  called `tz` or `zoneinfo`) maintained by IANA.
+: A set of rules representing the relationship of local time to UTC
+  for a particular place or region. Mathematically, a time zone can 
+  be thought of as a function that maps timestamps to UTC offsets.
```

Comment:
I'm using "timestamp" here to mean the same as it does in RFC 3339:

>       Timestamp   This term is used in this document to refer to an
>                  unambiguous representation of some instant in time.

In the context of that definition, there's no ambiguity. So I don't think there's a need to qualify "timestamp" here, other than perhaps to clarify that we're using the same definition as RFC 3339.

It's a separate and valid discussion of whether there should be a standardized string format that omits the offset. Such a string format would indeed be ambiguous because there are potentially multiple matching offsets, e.g. `2020-03-08T01:30[America/Los_Angeles]` could map to `2020-03-08T01:30-07:00[America/Los_Angeles]` or `2020-03-08T01:30-08:00[America/Los_Angeles]`.

Note that an offset-less format is out of scope to this document and this PR, but hopefully is something that we'll consider at a later date for a different specification.